users/tests.py

```diff
@@ -1,3 +1,90 @@
 from django.test import TestCase
+from users.models import CustomUser
+from django.urls import reverse
+from rest_framework.test import APITestCase
+from rest_framework.test import APIClient
 
-# Create your tests here.
+
+class TestCustomUserLogin(TestCase):
+    def setUp(self):
+        user = CustomUser.objects._create_user(
+            "testuser@example.com",
+            "testuserpasswd",
+            False,
+            False,
+        )
+
+    def test_that_user_should_login_correctly(self):
```

Comment:
I know that it sounds much more naturally if you use "that" in tests. Sounds like in Polish :)
But you can just skip it. Don't worry, it will look still good after **that** operation ;)

Reply:
Done